File: Python/twoSum.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        if len(nums) <= 1:
            return False
        dict = {}
        for i in range(len(nums)):
            if target - nums[i] in dict:
                return [dict[target - nums[i]], i]
            else:
                dict[nums[i]] = i

File: C++/twoSum.cpp
class Solution
{
public:
	vector<int> twoSum(vector<int>& nums, int target) {
		unordered_map<int, int> help;
		vector<int> result;
		for (int i = 0; i < nums.size(); ++i) {
			int numberToFind = target - nums[i];

			if (help.find(numberToFind) != help.end()) {
				result.push_back(help[numberToFind]);
				result.push_back(i);
			}
			help[nums[i]] = i;
		}
		return result;
	}
};